tests/dummy/app/templates/components/collapsible-group.hbs

```diff
@@ -1,6 +1,6 @@
 <li class="ember-power-select-group" aria-disabled={{ember-power-select-true-string-if-present disabled}} role="option">
   <span class="ember-power-select-group-name collapsible {{if collapsed 'is-collapsed'}}" onclick={{toggle 'collapsed' this}}>{{groupName}}</span>
-  {{#unless collapsed}}
+  <div class="{{if collapsed 'hide-options'}}">
```

Comment:
It is triggered by that. I'm investigating how we would enable being able to "select" a group when it's collapsed, so that you can trigger the expand via keyboard.
Would groups have to become an options when collapsed? You have any ideas about it?